packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -173,24 +177,35 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
         <div
           style={maxRows != null && tagState.showCollapseButton && isCollapsed ? {maxHeight: tagState.maxHeight, overflow: 'hidden'} : undefined}
           ref={containerRef}
-          className={classNames(styles, 'spectrum-Tags-container')}>
-          <div
-            ref={tagsRef}
-            {...gridProps}
-            className={classNames(styles, 'spectrum-Tags')}>
-            {visibleTags.map(item => (
-              <Tag
-                {...item.props}
-                key={item.key}
-                item={item}
-                state={state}
-                allowsRemoving={allowsRemoving}
-                onRemove={onRemove}>
-                {item.rendered}
-              </Tag>
-            ))}
-          </div>
-          {showActions &&
+          className={
+            classNames(
+              styles,
+              'spectrum-Tags-container',
+              {
+                'spectrum-Tags-container--empty': isEmpty
+              }
+            )
+          }>
+          <FocusRing focusRingClass={classNames(styles, 'focus-ring')}>
```

Comment:
Not sure the FocusRing makes sense when ShowAll hides all items. The hiding all items in this story is an existing bug. Would this go away if we fix that bug and always show one tag?